v1/src/main/java/com/google/cloud/teleport/spanner/ImportPipeline.java

```diff
@@ -204,6 +204,17 @@ public interface Options extends PipelineOptions {
     ValueProvider<RpcPriority> getSpannerPriority();
 
     void setSpannerPriority(ValueProvider<RpcPriority> value);
+
+    @TemplateParameter.Integer(
+        order = 12,
+        optional = true,
+        description = "Early Index Create Threshold",
+        helpText =
+            "The threshold for the number of indexes and foreign keys that determines whether to create indexes before data loading. If the total number of indexes and foreign keys is less than this threshold, they will be created before data import for better performance. The default value is 40.")
```

Comment:
from the code below it sounds the implementation is the opposite,  index created early when larger than the threshold

Reply:
Good catch. Fixed it.